src/libs/actions/PaymentMethods.js

```diff
@@ -114,10 +116,68 @@ function clearDebitCardFormErrorAndSubmit() {
     });
 }
 
+/**
+ * Call the API to transfer wallet balance.
+ * @param {Object} paymentMethod
+ * @param {String|Number} paymentMethod.methodID
+ * @param {'bankAccount'|'debitCard'} paymentMethod.type
```

Comment:
NAB, but these docs are inconsistent with how we do things. If there are multiple types we use `*` and the last one should be `String`. I will try to propose a style guide change so it doesn't need to be mentioned.